frontend/app/(footer)/event/create/page.tsx

```diff
@@ -220,3 +229,6 @@ export default function CreateEvent() {
     </div>
   );
 }
+function uuid() {
```

Comment:
??

Reply:
wtf idk how that got in there lol i mightve quick fixed it